kura/distrib/RELEASE_NOTES.txt

```diff
@@ -1,550 +1,213 @@
-Eclipse Kura - 4.1.1   - May 2020
--------------------------------------------------------------------------
+Eclipse Kura - 5.4.0-SNAPSHOT - November 2023
+-------------------------------------------------------------------------------------------------
 Description:
-Eclipse Kura 4.1.1 is a service release to address various discovered bugs and functional issues.
+This minor release of Eclipse Kura, compatible with Java 8 and OSGi R7, introduces support for IPv6 in new networking and generic profiles, WPA-enterprise client integration, an all new set of REST APIs to expose the framework services to simplify the integration of new and modern web UIs. Eclipse Kura 5.4.0 improves the generic profiles with a new method for interface listing and identification, and an improved system for RAM assignment at installation phase based on the available RAM in the target system. Finally, additional changes and improvements in the northbound connectivity connector for Eclipse Kapua with an improved management of APP messages and birth messages published with QoS 1.
```

Comment:
WPA-enterprise is also exclusive to the new networking, isn't it?

Reply:
I used what you wrote [here](https://projects.eclipse.org/projects/iot.kura/releases/5.4.0) as reference. I'll update it to make it more clear

Reply:
Maybe we need to update both, then

Reply:
Addressed in a29c301d91395e1ddd0043533ca4c32b80b817c9